lang/en/docs/cli/unlink.md

```diff
@@ -4,9 +4,11 @@ guide: docs_cli
 layout: guide
 ---
 
+{% include vars.html %}
+
 <p class="lead">Unlink a previously created symlink for a package.</p>
 
-To remove a symlinked package created with [`yarn link`](./link), `yarn unlink` can be used.
+To remove a symlinked package created with [`yarn link`]({{url_base}}/docs/cli/link), `yarn unlink` can be used.
```

Comment:
Yep, that’s more consistent with the others (while the previous version also worked)